tickle

```diff
@@ -1 +1 @@
-1
+Fri, Sep 25, 2020 7:19:00 AM
```

Comment:
Did you mean to include this?

Reply:
If a previous build is not available for the current state of the code, this file is used for release build automation to start a new build without making any functional changes.  This is useful in cases where the previous build was deleted.